scripts/packages/postremove.sh

```diff
@@ -1,21 +1,68 @@
 #!/bin/sh
+
 # Determine OS platform
 # shellcheck source=/dev/null
 . /etc/os-release
 
-if [ "$ID" = "freebsd" ]; then
-    echo "Stop and remove nginx-agent service"
+_stop_agent_freebsd() {
```

Comment:
Could you remove the underscore prefix from these function names so that it is consistent with the naming convention in other script.

Reply:
@dhurley done in https://github.com/nginx/agent/pull/352/commits/9bb401d7571bf798aeb6607b5d5b439de51f7364